remove all final mentions of bionemo framework2 (#1652)
NVIDIA/bionemo-framework: docs/docs/main/getting-started/pre-reqs.md

# Hardware and Software Prerequisites for BioNeMo Recipes

Before you begin using the BioNeMo Recipes, ensure the hardware and software prerequisites outlined below are
met.

## Hardware Prerequisites

> The BioNeMo Recipes is compatible with environments that have access to NVIDIA GPUs. `bfloat16` precision requires an

Reviewer: ```suggestion
BioNeMo Recipes are compatible with environments that have access to NVIDIA GPUs. `bfloat16` precision requires an
```

## Software Prerequisites

> The BioNeMo Recipes is supported on x86 Linux systems.

Reviewer: ```suggestion
BioNeMo Recipes are supported on x86 and arm Linux systems.
```
